src/storage/routing/RegexRouterRule.ts

```diff
@@ -6,6 +6,21 @@ import { trimTrailingSlashes } from '../../util/PathUtil';
 import type { ResourceStore } from '../ResourceStore';
 import { RouterRule } from './RouterRule';
 
+/**
+ * Utility class to easily configure Regex to ResourceStore mappings in the config files.
+ */
+export class RegexRule {
+  public constructor(
+    public readonly regex: string,
+    public readonly store: ResourceStore,
+  ) {}
+}
+
+interface Route {
+  regex: RegExp;
+  store: ResourceStore;
+}
```

Comment:
These can be combined in a single class if the regex conversion happens in the constructor.